src/main/java/com/example/webapp/Model/Department.java

```diff
@@ -0,0 +1,47 @@
+package com.example.webapp.Model;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import java.util.Objects;
+
+@Entity
+@Table
+@Getter
+@Setter
+@NoArgsConstructor
+public class Department {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+
+    private long department_id;
```

Comment:
camel case and add column annotation as you declared here. Applicable for all fields

Reply:
Done